website/docs/internals/debugging.md

```diff
@@ -0,0 +1,29 @@
+import Tags from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Debugging Sapling SCM
+
+## Logging
+
+Set the `ui.debug` setting to `true` to get debug logging of the Python
+interface surface.
```

Comment:
Please change `surface.` to `surface:` and then add an example so folks can copy/paste:

    ```
    sl config --user ui.debug true
    ```